src/hardware/input/mouse_interfaces.h

```diff
@@ -54,15 +53,15 @@ void MOUSEDOS_FinalizeInterrupt();
 
 void MOUSEDOS_NotifyMoved(const float x_rel, const float y_rel,
                           const uint32_t x_abs, const uint32_t y_abs);
-void MOUSEDOS_NotifyButton(const MouseButtons12S new_buttons_12S);
+void MOUSEDOS_NotifyButton(const MouseButtons12S in_buttons_12S);
```

Comment:
Yeah, so I dislike these `in_` prefixes everywhere. I suggest dropping them as they're confusing, or maybe use a better name that conveys intent. Maybe `buttons_state_something` in this case?

Reply:
Dropped the prefix from the interfaces, everywhere.